Dockerfile

```diff
@@ -31,10 +17,15 @@ RUN apt-get update \
     && python -m venv /opt/venv \
     && pip install --upgrade pip yq wheel \
     && pip install -r requirements.txt \
-    && ./kapitan/inputs/helm/build.sh \
-    && ./kapitan/dependency_manager/helm/build.sh \
     && pip install .
 
+# Install Helm
+RUN apt-get install --no-install-recommends -y curl \
+    && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
```

Comment:
should we be pinning the version of helm3 that gets downloaded? I believe if I'm reading the get_helm.sh script correctly, we could technically set a DESIRED_VERSION to control the helm3 version being downloaded instead of defaulting to latest.

Reply:
get_helm.sh supports `--version` flag. What would be the benefit of pinning the version ?

Reply:
I feel like in general there are bug fixes here and there that happen with any software. I feel like since we directly depend on the helm binary and we have an "open" interface with `helm_params` it might be worth being methodical on upgrading versions. Just a thought, don't think it's required.

Reply:
The get_helm.sh script is design to fetch only v3 version. So, all updates should be backward compatible.

Just like this image is using latest python3.7, git or gnupg binaries, I don't think it should cause issues using the latest helm 3 release.